src/time_travel/patchers/select_patcher.py

```diff
@@ -13,6 +14,8 @@ class SelectPatcher(BasicPatcher):
         - select.select
     """
     
+    EventTypes = Enum('select', 'READ WRITE EXCEPTIONAL')
```

Comment:
It's actually is a type:
https://docs.python.org/3/library/enum.html#functional-api